app/Notification/WebhookNotification.php

```diff
@@ -35,6 +35,10 @@ public function notifyUser(array $user, $event_name, array $event_data)
      */
     public function notifyProject(array $project, $event_name, array $event_data)
     {
+        if(!empty($_SERVER['HTTP_X_SKIP_WEBHOOKS']) && $_SERVER['HTTP_X_SKIP_WEBHOOKS']) {
```

Comment:
1. You should not use global variables. There is a class `Kanboard\Core\Http\Request` for that.
2. You should add tests
3. You should update the documentation if you add new features (folder `doc`)